src/js.cookie.js

```diff
@@ -67,7 +67,7 @@
 					attributes.expires && '; expires=' + attributes.expires.toUTCString(), // use expires attribute, max-age is not supported by IE
 					attributes.path    && '; path=' + attributes.path,
 					attributes.domain  && '; domain=' + attributes.domain,
-					attributes.secure  && '; secure'
+					(attributes.secure  && '; secure') || '' // use '' for falsy values, because [NaN, false, 0].join('') === 'NaNfalse0;
```

Comment:
Wouldn't it be the same for, say `domain` (if I happen to pass it to it)?

In that case we would clearly state that it'd be false usage and needed be fixed by them imo.